### DISCUSSION TITLE
Daily 19/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#208] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve con la conexion de Registración al backend y la verificación de los campos... Pude avanzar en parte pero no se llega a guardar en la base de datos. Por la tarde participé del encuentro con las profesoras de Proyecto Integrador en donde recibimos una devolución sobre los diagramas y sobre el trabajo en general. Hay que modificar los diagramas de clases, der y relacional, deben tener coherencia. Por la noche tuve clase de inglés.
Hoy continúo  con la registración, ya se validan todos los campo y se registra el usuario en la base de datos, pero ese usuario no puede ingresar mediante el login, así que sigo trabajando en eso. Participaré, además de la clase sincrónica de hoy.
Dificultades, los recursos de mi computadora son limitados, así que toda comprobación me lleva mucho mas tiempo.

**Comment by BraianTroncoso** [comment]
Ayer seguí trabajando en mis tareas, participé de la clase y mostré el proyecto funcionando. Participé de meets para ayudar a mis compañeros y solucionamos unos bugs. Sigo avanzando con otras tareas, y me voy asignando las que queden.
Dificultades, los recursos de mi computadora son limitados, así que toda comprobación me lleva mucho mas tiempo, sumado que entre el trabajo y otras responsabilidades saco tiempo de donde no hay para avanzar.
